uber/models.py

```diff
@@ -890,7 +1021,7 @@ def model_mixin(cls, model):
         for name in dir(model):
             if not name.startswith('_'):
                 attr = getattr(model, name)
-                if hasattr('target', '__table__') and name in target.__table__.c:
+                if hasattr(target, '__table__') and name in target.__table__.c:
```

Comment:
This change is unrelated to this PR, and I will revert it. I do want to revisit it however, so I will open an issue about it.